src/app/shared/components/filters-list/user-radius-set/user-radius-set.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, OnInit } from '@angular/core';
+import { ChangeContext } from '@angular-slider/ngx-slider';
+import { FormControl } from '@angular/forms';
+import { Store } from '@ngxs/store';
+import {
+  GetFilteredWorkshops,
+  SetRadiusSize
+} from '../../../store/filter.actions';
+import { DestroyableDirective } from '../../../directives/destroyable.directive';
+import { debounceTime, takeUntil } from 'rxjs/operators';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+@Component({
+  selector: 'app-user-radius-set',
+  templateUrl: './user-radius-set.component.html',
+  styleUrls: ['./user-radius-set.component.scss']
+})
+export class UserRadiusSetComponent
+  extends DestroyableDirective
+  implements OnInit
+{
+  public defaultValue: number = 5;
+  public options = { floor: 2, ceil: 10 };
+  public currentRadius: FormControl = new FormControl({
+    value: '',
+    disabled: true
+  });
+  8;
+  constructor(private store: Store, private snackBar: MatSnackBar) {
+    super();
+  }
+
+  ngOnInit(): void {
+    this.setValue(this.defaultValue);
+    this.subscribeOnValueChanges();
+  }
+
+  public radiusHandler(event: ChangeContext): void {
+    event.pointerType && this.setValue(event.highValue);
+  }
+
+  private setValue(number): void {
+    this.currentRadius.setValue(number);
+    this.store.dispatch(new SetRadiusSize(number));
+  }
+
+  private subscribeOnValueChanges() {
+    this.currentRadius.valueChanges
+      .pipe(takeUntil(this.destroy$), debounceTime(300))
+      .subscribe(() => {
+        this.store.dispatch(new GetFilteredWorkshops(true));
+        this.snackBar.open(
+          'Важливо! Тільки 100 найближчих гуртків будуть відображені на мапі.',
+          '',
+          {
+            duration: 5000,
+            panelClass: ['red-snackbar']
+          }
+        );
+      });
```

Comment:
I think you misunderstood the requirements, I don't think this message should appear each time the radius is changed, I think it should be a static display at the top of the page  (on init) with a "cross" icon, so it can be closed.
![image](https://user-images.githubusercontent.com/27493959/197976325-f67323e7-eb5c-4a10-95e0-7d864916142c.png)